### PR TITLE
[zkApps renaming]  graphQL, cli, snapps-test-tool

### PR DESCRIPTION
**Pull request by deepthiskumar**
Checklist:

- [ ] Modified the current draft of release notes with details on what is completed or incomplete within this project
- [ ] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them

* Closes #0000

### CODE REVIEW

**Review by mitschabaude — APPROVED** [review]

File: src/app/cli/src/init/client.ml [hunk 1]
```diff
@@ -1045,7 +1045,7 @@ let pooled_snapp_commands =
              let kvs = Yojson.Safe.Util.to_assoc raw_response in
              List.hd_exn kvs |> snd |> return
            with _ ->
-             eprintf "Failed to read result of pooled snapp commands" ;
+             eprintf "Failed to read result of pooled zkapp commands" ;
```
Comment (psteckler) [inline src/app/cli/src/init/client.ml:1048]: should this be `zkApp`?

File: src/app/test_executive/test_common.ml [hunk 2]
```diff
@@ -7,24 +7,24 @@ open Integration_test_lib
 module Make (Inputs : Intf.Test.Inputs_intf) = struct
   open Inputs.Engine
 
-  let send_snapp ~logger node parties =
-    [%log info] "Sending snapp"
+  let send_zkapp ~logger node parties =
+    [%log info] "Sending zkapp"
```
Comment (psteckler) [inline src/app/test_executive/test_common.ml:11]: `zkApp`?

File: src/app/test_executive/test_common.ml [hunk 3]
```diff
@@ -7,24 +7,24 @@ open Integration_test_lib
...
-    match%bind.Deferred Network.Node.send_snapp ~logger node ~parties with
-    | Ok _snapp_id ->
+    match%bind.Deferred Network.Node.send_zkapp ~logger node ~parties with
+    | Ok _zkapp_id ->
         [%log info] "Snapps transaction sent" ;
         Malleable_error.return ()
     | Error err ->
         let err_str = Error.to_string_mach err in
-        [%log error] "Error sending snapp"
+        [%log error] "Error sending zkapp"
```
Comment (psteckler) [inline src/app/test_executive/test_common.ml:19]: ditto

File: src/app/test_executive/test_common.ml [hunk 4]
```diff
@@ -37,7 +37,7 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
           Malleable_error.return () )
         else (
           [%log error]
-            "Error sending snapp, for a reason other than the expected \"%s\""
+            "Error sending zkapp, for a reason other than the expected \"%s\""
```
Comment (psteckler) [inline src/app/test_executive/test_common.ml:40]: ditto

File: src/app/test_executive/zkapps.ml [hunk 5]
```diff
@@ -361,7 +361,7 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
     in
     let%bind () =
       section "Send a snapp to create snapp accounts"
```
Comment (psteckler) [inline src/app/test_executive/zkapps.ml:363]: did you want to update the section text, "Send a zkApp ...`?

File: src/app/zkapp_test_transaction/lib/commands.ml [hunk 6]
```diff
@@ -139,10 +139,10 @@ let generate_snapp_txn (keypair : Signature_lib.Keypair.t) (ledger : Ledger.t)
     Transaction_snark.For_tests.create_trivial_predicate_snapp
       ~constraint_constants ~protocol_state_predicate spec ledger ~snapp_kp
   in
-  printf "Snapp transaction yojson: %s\n\n%!"
+  printf "Zkapp transaction yojson: %s\n\n%!"
```
Comment (psteckler) [inline src/app/zkapp_test_transaction/lib/commands.ml:142]: `zkApp`?

File: src/app/zkapp_test_transaction/lib/commands.ml [hunk 7]
```diff
@@ -139,10 +139,10 @@ let generate_snapp_txn (keypair : Signature_lib.Keypair.t) (ledger : Ledger.t)
...
     (Parties.to_yojson parties |> Yojson.Safe.to_string) ;
-  printf "(Snapp transaction graphQL input %s\n\n%!"
-    (graphql_snapp_command parties) ;
+  printf "(Zkapp transaction graphQL input %s\n\n%!"
```
Comment (psteckler) [inline src/app/zkapp_test_transaction/lib/commands.ml:144]: ditto

**Review by mrmr1993 — APPROVED** [review]

**Pull request merged**